packages/react/src/evm/providers/wallet-ui-states-provider.tsx

```diff
@@ -20,6 +20,7 @@ export type ModalConfig = {
     onLogin?: (token: string) => void;
     onLogout?: () => void;
   };
+  isEmbed?: boolean;
```

Comment:
so the API is

`<ConnectWallet isEmbed=true />` ?

Reply:
No - It's just `<ConnectEmbed />`

Reply:
This is internal code